python/langsmith/schemas.py

```diff
@@ -63,8 +63,22 @@ def my_function(bar: int, my_val: Attachment):
     data: bytes
 
 
-Attachments = Dict[str, Union[Tuple[str, bytes], Attachment]]
-"""Attachments associated with the run. Each entry is a tuple of (mime_type, bytes)."""
+Attachments = Dict[str, Union[Tuple[str, bytes], Attachment, str]]
```

Comment:
A couple comments:

This should be:
```
Attachments = Dict[str, Union[Tuple[str, bytes], Attachment, Tuple[str, Path]]
```

However we need to make sure this updated schema does not break existing code.